WORKSPACE

```diff
@@ -30,11 +28,11 @@ maven_install(
         "androidx.appcompat:appcompat:1.2.0",
         "com.google.android.material:material:1.2.1",
         "androidx.activity:activity-compose:1.3.0-alpha02",
-        "androidx.compose.material:material:{}".format(_JETPACK_COMPOSE_VERSION),
```

Comment:
Also might want to update:

core - 1.5.0
appcompat - 1.3.0
material - i think you can just delete it
activity-compose: 1.3.0-beta01

IIRC there were some deps that we needed to update that impacted runtime behavior (like I think if you don't update activity-compose it will crash the app on startup or something). I don't think it impacted compile time behavior tho.